docs/tutorials/11_GMPHDTutorial.py

```diff
@@ -0,0 +1,597 @@
+
+    # Plot the contents of these lists on the right axis
+    if new_truths:
+        truths.set_offsets(new_truths)
+    if new_tracks:
+        tracks.set_offsets(new_tracks)
+    if new_measurements:
+        measurements.set_offsets(new_measurements)
+    if new_clutter:
+        clutter.set_offsets(new_clutter)
+
+    # Create a legend. The use of Line2D is purely for the visual in the legend
+    data_types = [Line2D([0], [0], color='white', marker='o', markerfacecolor='blue', markersize=15,
+                         label='Ground Truth'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='orange', markersize=15,
+                         label='Clutter'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='green', markersize=15,
+                         label='Detection'),
+                 Line2D([0], [0], color='white', marker='o', markerfacecolor='red', markersize=15,
+                         label='Track')]
+    axR.legend(handles=data_types, bbox_to_anchor=(1.0, 1), loc='upper left')
+
+    return sf, truths, tracks, measurements, clutter
+
+# Set up the x, y, and z space for the 3D axis
+xx = np.linspace(x_min-5, x_max+5, 100)
+yy = np.linspace(y_min-5, y_max+5, 100)
+x, y = np.meshgrid(xx, yy)
+zarray = np.zeros((100, 100, number_steps))
+
+# Create the matplotlib figure and axes. Here we will have two axes being animated in sync.
+# `axL` will be the a 3D axis showing the Gaussian mixture
+# `axR` will be be a 2D axis showing the ground truth, detections, and updated tracks at 
+# each time step. 
+fig = plt.figure(figsize=(16, 8))
+axL = fig.add_subplot(121, projection='3d')
+axR = fig.add_subplot(122)
+axR.set_xlim(x_min-5, x_max+5)
+axR.set_ylim(y_min-5, y_max+5)
+
+# Add an initial surface to the left axis and scattered points on the right axis. Doing
+# this now means that in the animate() function we only have to update these variables
+sf = axL.plot_surface(x, y, zarray[:, :, 0], cmap=cm.RdBu, linewidth=0, antialiased=False)
+truths = axR.scatter(x_min-10, y_min-10, c='blue', linewidth=6, zorder=0.5)
+tracks = axR.scatter(x_min-10, y_min-10, c='red', linewidth=4, zorder=1)
+measurements = axR.scatter(x_min-10, y_min-10, c='green', linewidth=4, zorder=0.5)
+clutter = axR.scatter(x_min-10, y_min-10, c='orange', linewidth=4, zorder=0.5)
+
+# Create and display the animation
+from matplotlib import rc
+anim = animation.FuncAnimation(fig, animate, frames=number_steps, interval=500,
+                               fargs=(sf, truths, tracks, measurements, clutter), blit=False)
+rc('animation', html='jshtml')
+anim
+
+# sphinx_gallery_thumbnail_number = 4
+
+# %%
+# References
+# ----------
+# .. [#] B. -. Vo and W. -. Ma, "The Gaussian Mixture Probability Hypothesis Density Filter," in IEEE Transactions on Signal Processing, vol. 54, no. 11, pp. 4091-4104, Nov. 2006, doi: 10.1109/TSP.2006.881190.
```

Comment:
@nperree-dstl does this conflict with the suggestions you made previously? I believe your suggestions changed the [1] to [#] for the in-text citations so that Sphinx could auto-generate the citations. Do you know if @sglvladi's suggestion here will work? Thank you for your help!

Reply:
I tried building the documentation locally for both methods, and only @sglvladi 's method allowed me to reference the same source multiple times. So I think I will go with that. Thank you! :)